usr/src/test/util-tests/runfiles/default.run

```diff
@@ -35,7 +35,6 @@ outputdir = /var/tmp/test_results
 [/opt/util-tests/tests/libjedec_test]
 [/opt/util-tests/tests/smbios]
 
-[/opt/util-tests/tests/show-overlay-exit]
```

Comment:
This file needs copyright update

Reply:
Because this was a mismerge, I didn't think it did.

Reply:
This file only changed due to earlier mismerge, no copyright change necessary.